docs/marketplace/listing.rst

```diff
@@ -27,6 +27,7 @@ Here's an example composer.json file:
         "description": "Example Plugin",
         "type": "mautic-plugin",
         "version": "1.0",
```

Comment:
```suggestion
```
I think we do not need to specify the version here either - had discussed with @dennisameling last week but didn't get around to making the PR!